dev-libs/libixion/libixion-9999.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=7
 
 PYTHON_COMPAT=( python3_{6,7,8} )
-inherit python-single-r1
+inherit python-single-r1 autotools
```

Comment:
autotools shouldn't be required for release package, why did you change it?

Reply:
based on a wrong assumption. Builds without autools for releases as expected, fixed locally.